ckanext/dcatapchharvest/profiles.py

```diff
@@ -1035,6 +1042,20 @@ def graph_from_dataset(self, dataset_dict, dataset_ref):  # noqa
                 if uri:
                     g.add((distribution, DCT.language, URIRef(uri)))
 
+            # Language
+            languages = resource_dict.get('language', [])
+            for lang in languages:
+                if lang.startswith('http://') or lang.startswith('https://'):
+                    # lang is already a full IRI, use directly
+                    g.add((distribution, DCT.language, URIRef(lang)))
+                else:
+                    uri = language_uri_map.get(lang)
```

Comment:
I haven't been clear enough, sorry: This line would fail, when we call `get(lang)` and the `lang`-attribute can't be found, right?
If I understand this correctly, you would need something like `get(lang, None)` and the rest can remain as it is, does that make sense?

Reply:
Ja, das finde ich auch gut. Ja, wenn du den Fallback noch mit reinnehmen kannst, bin ich hier zufrieden 😍

Reply:
done